K2Bridge.Tests.UnitTests/Visitors/AggregationVisitorTests.cs

```diff
@@ -37,24 +37,6 @@ public string AggregationVisit_WithAvgAgg_ReturnsAvgPrimary()
             return aggregateClause.KustoQL;
         }
 
-        [TestCase(ExpectedResult = "['B']=avg(fieldB), ['A']=avg(fieldA)")]
```

Comment:
Nota bene: I removed the test cases where a metric aggregation had sub aggregations; this does not happen in practice:

https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket.html

> Bucket aggregations, **as opposed to metrics aggregations**, can hold sub-aggregations.